oxford-metadata.ttl

```diff
@@ -246,6 +246,14 @@
     a :Concentration ;
     rdfs:label "Extracellular ionic concentrations" .
 
+:ExtracellularBathConcentration
+    a :Concentration ;
```

Comment:
I can't find the little button for suggest changes directly, but I was imagining these two would both have
```
a: Class ;
a: ExtracellularConcentration ;
```
rather than
```
a: Concentration ;
```

Reply:
I don't understand how the `Class` thing works. Why do some of the other annotations above not have it? E.g.

```
:CytosolicConcentration
    a :Concentration ;
    rdfs:label "Cytosolic ionic concentrations" .

:ExtracellularConcentration
    a :Concentration ;
    rdfs:label "Extracellular ionic concentrations" .
```

And what's the difference between a category and a class?

Reply:
I think we need some comments at the top of the file explaining how the hierarchy works

Reply:
We defined Category :-), you are right it should have been a Category I think

```
:Category
    a rdfs:Class ;
    rdfs:label "A category of annotations" ;
    rdfs:comment "A class for classes that contain annotations, used to build a tree hierarchy in the annotation UI." .
```

Reply:
But @jonc125  can come to our rescue here. Does it show all the tree structure Jon or is it just things annotated with Category ?!

Reply:
`rdfs:Class` just means 'things that contain other things'. I'm not sure if it has an explicit impact in weblab-fc at present.

`:Category` is used in the annotation tool. Anything that either explicitly is-a `:Category` or transitively is-a something that is-a `:Category` appears in the tree, and any term that both is-a (explicit or transitive) `:Category` *and* is-a (explicit or transitive) `:Annotation` can be used to annotate a model variable.

So you don't want a term that is used to annotate a model variable to be a `:Category` as well, but it could be a `rdfs:Class` to indicate that it's *also* the parent of other terms. This might not be needed though.

Reply:
OK, so we don't need to say ExtracellularBathConcentration it is a `Category` because it inherits that info from `Concentration` via `ExtracellularConcentration`. Cheers Jon!

Reply:
So... the current PR is ok then? (once I run the ttl2rdf)

Reply:
I think so, I'm happy with that for now. We can worry about more sophisticated ion ontology stuff once the protocols can cope with it I think.